pkg/util/util.go

```diff
@@ -50,3 +52,26 @@ func Decode(buffer io.Reader, byteOrder binary.ByteOrder, outputs ...interface{}
 	}
 	return nil
 }
+
+// ResolveDNSAddress gets ip:port or dns:port as input and resolves the dns name if exists
```

Comment:
>ResolveDNSAddress takes address string in the format of host:port as the input; "host" can be either IP or DNS name. If a DNS name is provided, it resolves the IP address. Its output is always in the format of IP:port. 

Do we need to do this IPv6 parsing here? Dont we do this in Antrea?

Reply:
We can resolve the DNS name here and remove the code https://github.com/vmware-tanzu/antrea/blob/main/pkg/agent/flowexporter/exporter/exporter.go#L178 as you mentioned.